core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -536,11 +536,54 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey WEB_CORS_ALLOW_CREDENTIAL =
```

Comment:
Please give some description for these batch of config. As this PR is a little big, give more information will be easy to review, thanks

Reply:
The purpose is to make cors origin, methods and headers configurable. FYI https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#the_http_response_headers

Reply:
Could you create a new ticket? So that we can review this PR focus on worker stacks page.

Reply:
@maobaolong Ok, I submit a new PR #14562 .